pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -153,6 +157,7 @@ def get_unit_data(self, unit):
             'email_md5': None,
             'source': unit.source,
             'target': unit.target,
+            'submitted_on': make_aware(datetime.now()),
```

Comment:
Agree with @julen, we want times only when we know they are valid.  We can't tell when a unit was actually translated when we import a file, thus the UI when we don't know the time should apply to almost any file that can't give us a unit translation time (for now I think we can just assume that none can give that).